src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2140,6 +2140,17 @@ void processContextElement
 
   while (cursor->more())
   {
+    if (strcasecmp(action.c_str(), "appendonly") == 0)
```

Comment:
I don't see it really like that ... We're talking entities here, and either it exists of it doesn't, regardless of its attributes.
However, this is a definition that must be decided upon.